Dockerfile

```diff
@@ -105,10 +105,16 @@ VOLUME ${ZB_HOME}/data
 RUN groupadd -g 1000 zeebe && \
     adduser -u 1000 zeebe --system --ingroup zeebe && \
     chmod g=u /etc/passwd && \
-    chown 1000:0 ${ZB_HOME} && \
-    chmod 0775 ${ZB_HOME} && \
+    chown -R 1000:0 ${ZB_HOME} && \
+    chmod -R 0775 ${ZB_HOME} && \
+# These directories are potentially mounted by users, eagerly creating them and setting ownership
+# to the zeebe user makes sure no permissions issues occur due to default volume ownership.
     mkdir ${ZB_HOME}/data && \
-    chmod 0775 ${ZB_HOME}/data
+    chown 1000:0 ${ZB_HOME}/data && \
+    chmod 0775 ${ZB_HOME}/data && \
+    mkdir ${ZB_HOME}/logs && \
+    chown 1000:0 ${ZB_HOME}/logs && \
+    chmod 0775 ${ZB_HOME}/logs
```

Comment:
:upside_down_face:  Changing the ordering to
```
mkdir ${ZB_HOME}/data
mkdir ${ZB_HOME}/logs
chown -R 1000:0 ${ZB_HOME}
chmod -R 0775 ${ZB_HOME}
```
reduces some duplication and makes it easier to read I think